extractor/first_and_last_segment_of_way.hpp

```diff
@@ -58,19 +59,15 @@ struct FirstAndLastSegmentOfWay
 
     static FirstAndLastSegmentOfWay min_value()
     {
-        return {std::numeric_limits<EdgeID>::min(),
-                std::numeric_limits<NodeID>::min(),
-                std::numeric_limits<NodeID>::min(),
-                std::numeric_limits<NodeID>::min(),
-                std::numeric_limits<NodeID>::min()};
+        return {OSMWayID(0),NodeID64(0),NodeID64(0),NodeID64(0),NodeID64(0)};
     }
     static FirstAndLastSegmentOfWay max_value()
     {
-        return {std::numeric_limits<EdgeID>::max(),
-                std::numeric_limits<NodeID>::max(),
-                std::numeric_limits<NodeID>::max(),
-                std::numeric_limits<NodeID>::max(),
-                std::numeric_limits<NodeID>::max()};
+        return {INVALID_OSM_WAYID,
+                INVALID_NODEID64,
+                INVALID_NODEID64,
+                INVALID_NODEID64,
+                INVALID_NODEID64};
     }
```

Comment:
as above, would prefer an additional definition for MAX_ID or keep usage of numeric_limits. INVALID just looks wrong to me in a max statement.